image-builder/image_builder.sh

```diff
@@ -279,11 +281,27 @@ create_rootfs_disk
 info "rootfs size ${ROOTFS_SIZE} MB"
 info "Copying content from rootfs to root partition"
 cp -a "${ROOTFS}"/* ${MOUNT_DIR}
+sync
 OK "rootfs copied"
 
 unmount
 # Optimize
 fsck.ext4 -D -y "${DEVICE}p1"
 detach
 
+info "Set device namespace information (metadata)"
+# Fill out namespace information
+tmp_img="$(mktemp)"
+chmod 0644 "${tmp_img}"
+# metadate header
+dd if=/dev/zero of="${tmp_img}" bs="${IMG_HEADER_SZ}M" count=1
+# append image data (rootfs)
+dd if="${IMAGE}" of="${tmp_img}" oflag=append conv=notrunc
+# copy final image
+mv "${tmp_img}" "${IMAGE}"
+# Set metadata header
+gcc -O2 "${script_dir}/nsdax.gpl.c" -o "${script_dir}/nsdax"
```

Comment:
Not necesary for this PR but I wonder if could create a separate makefile target for this,  and would be really useful would be 

```
make 
make install DESDIR
```

then  check in this scritpt if this binary exist, it does run it.

I ask because fedora is packaging osbuilder scripts today. So I dont want to compile this at post install.

Reply:
Thanks for add the issue.